src/main/java/com/google/devtools/build/lib/syntax/BUILD

```diff
@@ -157,5 +157,5 @@ cc_binary(
         "//conditions:default": ["cpu_profiler_unimpl.cc"],
     }),
     linkshared = 1,
-    deps = ["@bazel_tools//tools/jdk:jni"],
```

Comment:
What's the explanation for this change? 🤔 It looks fine to me, just wondering how it's related to this change.

Reply:
I mentioned above, when I rebased I wasn't actually able to build Bazel master! Very strange but this fixed the issue... I'd be interested to hear from someone if this was a correct solution to the problem as this target DOES exist within Bazel.

Reply:
@Qinusty Sorry, I didn't see that! The `@bazel_tools` reference is a reference to a repository that's built into each Bazel binary. So, the contents of that repository change depending on which Bazel version you're using. I think what happened was that when you tried to build Bazel master and it failed, you were using a Bazel version that didn't have that target (`@bazel_tools//tools/jdk:jni`) yet.

Does it work with the latest Bazel release now? If yes... maybe that was just a temporary hiccup (we only test that you can build Bazel master branch with the latest stable release that was available at the time of the commit). If it still doesn't work without that change, it's pretty weird.

Reply:
Ah that is likely the case, I'll drop the commit rebuild and push if successful!

Reply:
Success!

Reply:
Great, thanks for checking! I'll revert that part then. :)